app.json

```diff
@@ -8,6 +8,7 @@
   "env": {
     "BUILDPACK_URL": "https://github.com/RocketChat/heroku-buildpack-meteor.git",
     "ROOT_URL": {
+			"value": "https://YOUR-APP-NAME.herokuapp.com"
```

Comment:
done and done. added required: true -- might as well assume to user will make that mistake too :)